blocks/tabs/tabs.css

```diff
@@ -1,58 +1,174 @@
 .tabs .tabs-list {
-    display: flex;
-    gap: 0.5ch;
+  display: flex;
+  gap: 0.5ch;
+  max-width: 100%;
+  font-size: var(--body-font-size-xs);
+  overflow-x: auto;
+}
+
+@media (width >= 600px) {
+  .tabs .tabs-list {
+    font-size: var(--body-font-size-s);
+  }
+}
+
+@media (width >= 900px) {
+  .tabs .tabs-list {
+    font-size: var(--body-font-size-m);
+  }
+}
+
+.tabs .tabs-list button {
+  flex: 0 0 max-content;
+  margin: 0;
+  border: 1px solid #dadada;
+  border-radius: 0;
+  padding: 0.5em;
+  background-color: var(--light-color);
+  color: initial;
+  font-weight: bold;
+  line-height: unset;
+  text-align: initial;
+  text-overflow: unset;
+  overflow: unset;
+  white-space: unset;
+  transition: background-color 0.2s;
+}
+
+.tabs .tabs-list button p {
+  margin: 0;
+}
+
+.tabs .tabs-list button[aria-selected='true'] {
+  border-bottom: 1px solid var(--background-color);
+  background-color: var(--background-color);
+  cursor: initial;
+}
+
+.tabs .tabs-panel {
+  margin-top: -1px;
+  padding: 24px;
+  border: 1px solid #dadada;
+  overflow: auto;
+}
+
+.tabs .tabs-panel[aria-hidden='true'] {
+  display: none;
+}
+
+/* Blue-White-Text Tab Style */
+.tabs.blue-white-text .tabs-list {
+  gap: 12px;
+  padding: 0;
+  margin-bottom: 0;
+  font-size: 14px;
+}
+
+.tabs.blue-white-text .tabs-tab,
+.tabs.silver-blue-text .tabs-tab {
```

Comment:
Wherever you are selecting for both tab colors like this, go ahead make the style the default for all tabs (since we haven't really established a default, we can start with this).
So in this case, just use 
`.tabs .tabs-tab {`